templates/security/forgot_password.html.twig

```diff
@@ -0,0 +1,7 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Forgot Password{% endblock %}
+
+{% block body %}
+    {{ form(form) }}
```

Comment:
well I hate this, but I suppose we improve later via the toolkit?

Reply:
Yep, this is just to get started.